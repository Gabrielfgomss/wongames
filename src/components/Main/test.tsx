import { render, screen } from "@testing-library/react"
import { Main } from "."

describe("<Main />", () => {
  it("should render the heading", () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole("heading", { name: /react avançado/i }),
    ).toBeInTheDocument()

    // Making snapshot
    // Server para verificar a arvore do DOM quando o teste é realizado para
    // ver se houve mudanças e elas estão corretas
    expect(container.firstChild).toMatchSnapshot()
  })
})
