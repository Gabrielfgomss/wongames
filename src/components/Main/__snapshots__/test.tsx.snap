// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
.c0 {
  background-color: red;
}

<main
  class="c0"
>
  <h1>
    React avan√ßado
  </h1>
</main>
`;
